core/templates/components/forms/schema-based-editors/schema-based-html-editor.component.ts

```diff
@@ -39,11 +39,11 @@ import { downgradeComponent } from '@angular/upgrade/static';
 
 export class SchemaBasedHtmlEditorComponent
 implements ControlValueAccessor, OnInit, Validator {
-  localValue: string;
-  @Input() disabled: boolean;
-  @Input() labelForFocusTarget: string;
-  @Input() uiConfig: unknown;
-  @Input() headersEnabled: boolean;
+  localValue!: string;
+  @Input() disabled!: boolean;
+  @Input() labelForFocusTarget!: string;
+  @Input() uiConfig!: unknown;
```

Comment:
Will look into this unknown later, need some discussion. So unable to replace this right now.